From: "Y.C. Chen" <yc_chen@aspeedtech.com>
To: dri-devel@lists.freedesktop.org
Cc: airlied@redhat.com, eich@suse.com, kuohsiang_chou@aspeedtech.com,
        "Y.C. Chen" <yc_chen@aspeedtech.com>
Date: Fri, 22 Aug 2014 11:00:17 +0800
Subject: [PATCH 1/2] drm/ast: Add missing entry to dclk_table[]

Signed-off-by: Egbert Eich <eich@suse.de>
Patch-Mainline: to be upstreamed

This avoid reading past the end of the list for certain modes

Signed-off-by: Y.C. Chen <yc_chen@aspeedtech.com>
---
 drivers/gpu/drm/ast/ast_tables.h |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/gpu/drm/ast/ast_tables.h
+++ b/drivers/gpu/drm/ast/ast_tables.h
@@ -99,6 +99,7 @@ static struct ast_vbios_dclk_info dclk_t
 	{0x25, 0x65, 0x80},					/* 16: VCLK88.75    */
 	{0x77, 0x58, 0x80},					/* 17: VCLK119      */
 	{0x32, 0x67, 0x80},				    /* 18: VCLK85_5     */
+	{0x6a, 0x6d, 0x80},					/* 19: VCLK97_75	*/
 };
 
 static struct ast_vbios_stdtable vbios_stdtable[] = {
